fieldbehavior/fieldbehavior.go

```diff
@@ -109,33 +109,72 @@ func isPresent(v protoreflect.Value, f protoreflect.FieldDescriptor) bool {
 }
 
 func clearFieldsWithBehaviors(m proto.Message, behaviorsToClear ...annotations.FieldBehavior) {
-	rangeFieldsWithBehaviors(m, func(
-		m protoreflect.Message,
-		f protoreflect.FieldDescriptor,
-		_ protoreflect.Value,
-		behaviors []annotations.FieldBehavior,
-	) bool {
-		if hasAnyBehavior(behaviors, behaviorsToClear) {
-			m.Clear(f)
-		}
-		return true
-	})
+	rangeFieldsWithBehaviors(
+		m.ProtoReflect(),
+		func(
+			m protoreflect.Message,
+			f protoreflect.FieldDescriptor,
+			_ protoreflect.Value,
+			behaviors []annotations.FieldBehavior,
+		) bool {
+			if hasAnyBehavior(behaviors, behaviorsToClear) {
+				m.Clear(f)
+			}
+			return true
+		},
+	)
 }
 
 func rangeFieldsWithBehaviors(
-	m proto.Message,
-	fn func(protoreflect.Message, protoreflect.FieldDescriptor, protoreflect.Value, []annotations.FieldBehavior) bool,
+	m protoreflect.Message,
+	fn func(
+		protoreflect.Message,
+		protoreflect.FieldDescriptor,
+		protoreflect.Value,
+		[]annotations.FieldBehavior,
+	) bool,
 ) {
-	d := m.ProtoReflect()
-	d.Range(func(f protoreflect.FieldDescriptor, v protoreflect.Value) bool {
-		if behaviors, ok := proto.GetExtension(
-			f.Options(),
-			annotations.E_FieldBehavior,
-		).([]annotations.FieldBehavior); ok {
-			return fn(d, f, v, behaviors)
-		}
-		return true
-	})
+	m.Range(
+		func(f protoreflect.FieldDescriptor, v protoreflect.Value) bool {
+			if behaviors, ok := proto.GetExtension(
+				f.Options(),
+				annotations.E_FieldBehavior,
+			).([]annotations.FieldBehavior); ok {
+				fn(m, f, v, behaviors)
+			}
+
```

Comment:
in general avoid blank lines within small-ish functions - different people prefer different amounts of blank lines and the easiest way to get it consistent is to avoid